lib/workers/repository/init/inherited.ts

```diff
@@ -99,5 +100,43 @@ export async function mergeInheritedConfig(
       'Removed global config from inherited config.',
     );
   }
-  return mergeChildConfig(config, filteredConfig);
+
+  if (is.nullOrUndefined(filteredConfig.extends)) {
+    return mergeChildConfig(config, filteredConfig);
+  }
+
+  let returnConfig = filteredConfig;
+
+  logger.debug('Resolving presets found in inherited config');
+  const resolvedConfig = await resolveConfigPresets(
+    returnConfig,
+    config,
+    config.ignorePresets,
+  );
+  logger.trace({ config: resolvedConfig }, 'Resolved inherited config');
+
+  const validationRes = await validateConfig('inherit', resolvedConfig);
+  if (validationRes.errors.length) {
+    logger.warn(
+      { errors: validationRes.errors },
+      'Found errors in presets inside the inherited configuration.',
+    );
+    throw new Error(CONFIG_VALIDATION);
+  }
+  if (validationRes.warnings.length) {
+    logger.warn(
+      { warnings: validationRes.warnings },
+      'Found warnings in presets inside the inherited configuration.',
+    );
+  }
```

Comment:
```suggestion
```
duplicate

Reply:
`validateConfig` returns both `errors` & `warnings` so it's not a duplicate.